cura/Arranging/Arrange.py

```diff
@@ -61,13 +64,17 @@ def create(cls, scene_root = None, fixed_nodes = None, scale = 0.5, x = 220, y =
 
         # If a build volume was set, add the disallowed areas
         if Arrange.build_volume:
-            disallowed_areas = Arrange.build_volume.getDisallowedAreas()
+            disallowed_areas = Arrange.build_volume.getDisallowedAreasNoBrim()
```

Comment:
Is there a reason for not simply altering the `getDisallowedAreas()` function? Seems not so intuitive to add a second method with almost the same name doing almost the same thing.

Reply:
As far as I can tell, this is the only place `getDisallowedAreas()` is used.

Reply:
It's used a few times in BuildVolume itself for showing which areas are disallowed. The annoying thing is that the *brim size setting is taken into account. Hmmmm maybe in the (near) future we can get rid of that as well.